src/controllers/legacycontrollersettings.h

```diff
@@ -500,8 +661,33 @@ class LegacyControllerFileSetting
         return QJSValue(stringify());
     }
 
+#ifdef MIXXX_USE_QML
+    void setValue(const QJSValue& rawValue) override {
+        QUrl path = rawValue.toVariant().value<QUrl>();
```

Comment:
We expect a string here, right? Not an QUrl object. 
If that's the case please avoid the variant roundtrip.

Reply:
Ok than please rename `rawValue` in `url`. Do we need error handling when not an URL type is passed? 
Can that happen?